.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+## Describe your changes
+<!--- Provide a general summary of your changes -->
+
+## Issue or discord link
+<!--- link relevant issues to this PR, or provide a link to a discord message/thread -->
+*
```

Comment:
extra `*`

Reply:
or is this intentional?

Reply:
It's intentional. Issues referenced in a bullet list will have the title automatically inserted, which is convenient